packages/grafana-e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,29 @@
+require('cypress-file-upload');
```

Comment:
Are you able to explain the changes tho these files - why they went from TS + ESM to JS + CJS?

Not objecting, just useful to know 👍🏻